apps/damap-frontend/src/app/components/layout/layout.component.html

```diff
@@ -136,6 +136,7 @@
             class="mat-focus-indicator"
             [title]="'layout.logo.text' | translate"
             [href]="'layout.logo.link' | translate"
+            target="_blank"
             rel="noopener">
```

Comment:
This line should be set to  rel="noopener noreferrer", to prevent phishing attacks. See [this link](https://www.freecodecamp.org/news/how-to-use-html-to-open-link-in-new-tab/#:~:text=as%20tabnabbing.-,What%20is%20tabnabbing%3F,-Tabnabbing%2C%20sometimes%20called) for more details.

Reply:
👍